Package.swift

```diff
@@ -215,7 +215,7 @@ let package = Package(
             dependencies: ["TSCTestSupport"]),
         .testTarget(
             name: "TSCUtilityTests",
-            dependencies: ["TSCUtility", "TSCTestSupport", "TSCTestSupportExecutable"]),
+            dependencies: ["TSCUtility", "TSCTestSupport", "SPMTestSupport", "TSCTestSupportExecutable"]),
```

Comment:
Do we need this since tests only run on macOS?

Reply:
Correct, this is not necessary anymore. Will remove it.